driver-core/src/main/java/com/datastax/driver/core/SessionManager.java

```diff
@@ -431,7 +437,7 @@ CloseFuture removePool(Host host) {
         }
 
         // Wait pool creation before removing, so we don't lose connectivity
-        ListenableFuture<?> allPoolsCreatedFuture = Futures.successfulAsList(poolCreatedFutures);
+        ListenableFuture<?> allPoolsCreatedFuture = Futures.allAsList(poolCreatedFutures);
```

Comment:
Now there is one, see [here](https://github.com/datastax/java-driver/blob/71c23f8b33dc5bd5cc66f3220ee119f1437b106e/driver-core/src/main/java/com/datastax/driver/core/SessionManager.java#L383-L384).